src/components/catchphrases/Catchphrases.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import '../controls/Controls';
+import './Catchphrases.css';
+
+export default function Catchphrases({ phrase }) {
+  return (
+    <div className="catchphrases-container">
+      <ul className="catchphrases">
+        {phrase.map((phrases) => {
```

Comment:
because then this would read ```phrases.map((phrase) => ``` which is a more accurate semantic representation of what's going on — you have an array of multiple phrases and you map through and for each phrase you will perform the callback function